wpengine.com.arecord.json

```diff
@@ -4,15 +4,14 @@
    "serviceId":"arecord",
    "serviceName":"WP Engine Digital Experience Platform",
    "version": 1,
-   "logoUrl":"https://wpengine.com/logo.png",
+   "logoUrl":"https://www.domainconnect.org/wp-content/uploads/2018/03/LGO-WPEngine-Horiz-reg-RGB-768x146.png",
    "description":"Configure ip address for the domain A record.",
    "syncBlock":false,
-   "syncPubKeyDomain": "landmark.wpengine.io",
+   "syncPubKeyDomain": "landmark.wpesvc.net",
    "syncRedirectDomain": "my.wpengine.com",
    "records":[
       {
          "type":"A",
-         "host":"@",
```

Comment:
Hi,
according to the spec _host_ field is required for A record

See: https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#53-template-record

> For each type, additional fields would be required.
> A: host, pointsTo, TTL